packages/esm-patient-list-app/src/patient-table/overflow-menu.component.tsx

```diff
@@ -0,0 +1,54 @@
+import { isDesktop, showToast, useLayoutType } from '@openmrs/esm-framework';
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { removePatientFromList } from '../api/api-remote';
+import { OverflowMenu, Layer, OverflowMenuItem } from '@carbon/react';
+import styles from './patient-table.scss';
+
+interface OverflowMenuCellProps {
+  cohortMembershipUuid: string;
+}
+
+const OverflowMenuComponent: React.FC<OverflowMenuCellProps> = ({ cohortMembershipUuid }) => {
+  const { t } = useTranslation();
+  const layoutType = useLayoutType();
+  const desktopLayout = isDesktop(layoutType);
+
+  const handleRemovePatientFromList = useCallback(() => {
+    removePatientFromList(cohortMembershipUuid)
+      .then(() =>
+        showToast({
+          title: t('removed', 'Removed'),
+          description: t('removePatientFromList', 'Successfully removed patient from list'),
+          kind: 'success',
+        }),
+      )
+      .catch(() =>
+        showToast({
+          title: t('error', 'Error'),
+          description: t('errorRemovePatientFromList', "Couldn't remove patient from this list"),
+          kind: 'error',
+        }),
+      );
+  }, [cohortMembershipUuid, t]);
+
+  return (
+    <Layer className={styles.layer}>
+      <OverflowMenu
+        data-floating-menu-container
+        ariaLabel="Remove patient from the list"
+        size={desktopLayout ? 'sm' : 'lg'}
+        flipped>
+        <OverflowMenuItem
```

Comment:
Add an ExtensionSlot in here, with slot name = "cohort-member-action-items"

Reply:
@vasharma05 Was the intention here to have the "Remove Patient" stuff as an extension?